integration-tests/WRITING_TESTS.md

```diff
@@ -0,0 +1,79 @@
+# Writing integration tests for snappy
```

Comment:
I think it's getting a little crowded in there. But if you think it's still readable, that's ok. I moved it.